README.md

```diff
@@ -15,7 +13,7 @@ The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator) enabl
 
 ### Usage
 
-[Refer](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to Simulate Scenario documentation to undersand how to Simulate and Reset Scenario.
+To Simulate Scenario documentation for a better understanding of the Simulate and Reset Scenario, refer to the content [here](docs/solution-pack-guide.md).
```

Comment:
To Simulate Scenario documentation for a better understanding of the Simulate and Reset Scenario, --> i still feel that something is correct here